extension/src/experiments/model/tree.ts

```diff
@@ -268,7 +268,14 @@ export class ExperimentsTree
 
   private getDescription() {
     const dvcRoots = this.experiments.getDvcRoots()
-    if (!definedAndNonEmpty(dvcRoots)) {
+
+    const total = sum(
+      dvcRoots.map(dvcRoot =>
+        this.experiments.getRepository(dvcRoot).getExperimentCount()
+      )
+    )
+
+    if (!total) {
```

Comment:
[F] If we have no experiments then all of the repos in the workspace are empty and we should not bother to show a description (previously `0 of 0 (max 7)` would have been shown).